jupyterhub/spawners.py

```diff
@@ -277,6 +277,9 @@ def get_pod_manifest(self):
             + "/"
             + options.get("project")
         )
+        git_host = urlparse(
+            os.environ.get("GITLAB_URL", "http://gitlab.renku.build"),
```

Comment:
I know we still have this elsewhere, but `http://gitlab.renku.build` as a default for development is deprecated. The code should fail if no `GITLAB_URL` is provided as environment variable.

Reply:
@ableuler I removed all of the instances of this wherever it appears in the whole repo. There were quite a few. I can roll them back without a problem if you did not want me to get them all.